domain/model/pokemon.go

```diff
@@ -0,0 +1,8 @@
+package model
+
+type Pokemon struct {
+	ID        int      `json:"id"`
+	Name      string    `json:"name"`
+}
+
+func (Pokemon) TableName() string { return "pokemons" }
```

Comment:
This doesn't look like it needs to be a method, you're not accessing the properties of the struct.
It could be a constant located on the package where you actually need it